src/main/java/seedu/clinic/MainApp.java

```diff
@@ -69,29 +74,50 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s clinic and {@code userPrefs}. <br>
+     * Returns a {@code ModelManager} with the data from {@code storage}'s clinic, user macros, and
+     * {@code userPrefs}. <br>
      * The data from the sample clinic will be used instead if {@code storage}'s clinic is not found,
      * or an empty clinic will be used instead if errors occur when reading {@code storage}'s clinic.
+     * An empty User Macros model will be used if {@code storage}'s User Macros are not found or if errors
+     * occur when reading {@code storage}'s User Macros.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         Optional<ReadOnlyClinic> clinicOptional;
-        ReadOnlyClinic initialData;
+        ReadOnlyClinic initialClinicData;
 
         try {
             clinicOptional = storage.readClinic();
-            if (!clinicOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample Clinic");
+            if (clinicOptional.isEmpty()) {
+                logger.info("Clinic data file not found. Will be starting with a sample Clinic");
             }
-            initialData = clinicOptional.orElseGet(SampleDataUtil::getSampleClinic);
+            initialClinicData = clinicOptional.orElseGet(SampleDataUtil::getSampleClinic);
         } catch (DataConversionException e) {
-            logger.warning("Data file not in the correct format. Will be starting with an empty Clinic");
-            initialData = new Clinic();
+            logger.warning("Clinic data file not in the correct format. Will be starting with an empty Clinic");
+            initialClinicData = new Clinic();
         } catch (IOException e) {
-            logger.warning("Problem while reading from the file. Will be starting with an empty Clinic");
-            initialData = new Clinic();
+            logger.warning("Problem while reading from the Clinic data file. Will be starting with an empty Clinic");
```

Comment:
Maybe "Problem encountered while reading..." or "Unable to read from..." would be better here

Reply:
True, will change it!